bindings/python/pymanifold.cpp

```diff
@@ -490,7 +500,25 @@ PYBIND11_MODULE(pymanifold, m) {
           "will result in geometrically valid output.\n"
           "\n"
           ":param circularSegments: Number of segments along its diameter. "
-          "Default is calculated by the static Defaults.");
+          "Default is calculated by the static Defaults.")
+      .def_property_readonly(
+          "data",
+          [](PolygonsWrapper &self) {
+            const Polygons &data = *self.polygons;
+            py::list polygon_list;
+            for (int i = 0; i < data.size(); ++i) {
+              py::list polygon;
+              for (int j = 0; j < data[i].size(); ++j) {
+                auto f = data[i][j];
+                py::tuple vertex = py::make_tuple(f[0], f[1]);
+                polygon.append(vertex);
+              }
+              polygon_list.append(polygon);
+            }
+            return polygon_list;
+          },
+          "Returns the vertices of the polygons as a "
+          "List[List[Tuple[float, float]]].");
```

Comment:
Thinking about it, it is not really needed as we can provide static functions instead of methods with this wrapper.

Reply:
> Why is this wrapper necessary?

You mean why did I add a .data member to the Polygons type? I could not find another way to get the data out. The question was asked here: https://github.com/elalish/manifold/discussions/439.

Reply:
> Thinking about it, it is not really needed as we can provide static functions instead of methods with this wrapper.

Sorry, I did not understand this.
Would you prefer a static function with similar functionality? E.g. `points = Polyogons.get_data(polygons_object)`.
Or did you mean something else?

Reply:
No, I think @elalish is asking about why this `Polygons` wrapper exists, and what I am saying is that we can remove this and use static `Extrude`/`Revolve` functions if we really want to (no opinion on this, just stating that this is possible).

Reply:
Agreed, that's exactly what I was thinking of.

Reply:
Ok, I'll fix.

Reply:
Wrapper removed!